onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -45,8 +45,8 @@
 # latency gain headers 
 trt_cuda_gain = 'TRT_CUDA_gain(%)'
 trt_cuda_fp16_gain = 'TRT_CUDA_fp16_gain(%)'
-trt_native_gain = 'EP_Native_TRT_gain(%)'
-trt_native_fp16_gain = 'EP_Native_TRT_fp16_gain(%)'
+trt_native_gain = 'EP_Standalone_TRT_gain(%)'
```

Comment:
i think the previous name was a bit more intuitive and more consistent with other header (trt_cuda)
or maybe we should do something like
trt_vs_cuda_gain  and trt_vs_native_gain or trt_vs_standalone_gain